install.md

```diff
@@ -5,18 +5,18 @@ To get started with OpenCost and Kubecost, **the recommended path is to [install
 
 <ins>Alternative install options:<ins>
 
-* You can also install directly with the [Kubecost helm chart](https://github.com/kubecost/cost-analyzer-helm-chart/) with helm 3 using the following commands. This provides the same functionality as the step above but doesn't generate a product token for managing tiers or upgrade trials. 
+* You can also install directly with the [Kubecost helm chart](https://github.com/kubecost/cost-analyzer-helm-chart/) with helm 3 using the following commands. This provides the same functionality as the step above but doesn't generate a product token for managing tiers or upgrade trials.
+
+```
+helm repo add kubecost https://kubecost.github.io/cost-analyzer/
+helm upgrade --install kubecost kubecost/cost-analyzer --namespace kubecost --create-namespace
+```
 
-   ```
```

Comment:
Did you want to remove the indentation here or is this a linter adjustment? I personally like it inline with the bullet point, but let me know what you think.

Reply:
With the indentation, the copy was adding spaces at the beginning of the second line when coping the commands. 

Agreed that the formatting looks nicer indented. 

See:
https://guide.kubecost.com/hc/en-us/articles/4407601821207-Installing-Kubecost

The copy works correctly on GH. 
Maybe we should see how this looks on guide and change back if it looks bad?